recipes-kernel/kernel-tests/kernel-test-nohz.bb

```diff
@@ -40,7 +40,7 @@ pkg_postinst_ontarget:${PN}-ptest:append() {
     ISOLATED_CPU=$((CPUS - 1))
 
     if [ $ISOLATED_CPU -gt 0 ]; then
-        echo 'set otherbootargs="${otherbootargs} isolcpus='$ISOLATED_CPU' nohz_full='$ISOLATED_CPU' mitigations=off intel_pstate=disable intel_idle.max_cstate=0 processor.max_cstate=0 nosoftlockup mce=ignore_ce audit=0 tsc=nowatchdog"' > /boot/runmode/no-hz-full-params.cfg
+        echo 'set otherbootargs="${otherbootargs} isolcpus=nohz,domain,managed_irq,'$ISOLATED_CPU' nohz_full='$ISOLATED_CPU' mitigations=off intel_pstate=disable intel_idle.max_cstate=0 processor.max_cstate=0 nosoftlockup mce=ignore_ce audit=0 tsc=nowatchdog"' > /boot/runmode/no-hz-full-params.cfg
```

Comment:
Would we want to mentions these params in [this readme](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/recipes-kernel/linux/nilrt-nohz/README.nohz#L44C55-L44C55)?
Also, for non-LV based applications [the instructions](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/recipes-kernel/linux/nilrt-nohz/README.nohz#L56) should be updated right?

Reply:
README.nohz updated.